fierce/fierce.py

```diff
@@ -333,7 +333,8 @@ def fierce(**kwargs):
     random_subdomain = str(random.randint(1e10, 1e11))
     random_domain = concatenate_subdomains(domain, [random_subdomain])
     wildcard = query(resolver, random_domain, record_type='A')
-    print("Wildcard: {}".format(wildcard[0].address if wildcard else "failure"))
+    wildcard_ips = set([rr.address for rr in wildcard.rrset]) if wildcard else None
```

Comment:
Argh, yes, that's what I get for last-minute changes. And good tip on the generator.

Reply:
How hard do you think it would be to add tests to cover this? I'm not familiar with mocking in Python, so it would be helpful to know if I'm about to go off into the weeds—but I'm happy to add tests if it's not a huge pain.

Reply:
Unfortunately we don't have a great way to test the behemoth `fierce` function right now. I wouldn't worry about the coverage.